linty_fresh/linters/xcodebuild.py

```diff
@@ -4,10 +4,12 @@
 
 from linty_fresh.problem import Problem
 
-XCODEBUILD_LINE_REGEX = re.compile(r'(?P<path>[^:]*):'
-                                   r'((?P<line>\d*)|(?P<reference>[^:]*)):'
-                                   r'(?:(?P<column>\d*):)?\s*(?P<level>\w*):'
-                                   r'\s*(?P<message>.*)')
+XCODEBUILD_LINE_REGEX = re.compile(
+    r'(?P<path>[^:]*):'
+    r'((?P<line>\d*)|(?P<reference>[^:]*)):'
+    r'(?:(?P<column>\d*):)?\s*(?P<level>((?i)error|warn(ing)?|note|info)):'
```

Comment:
would it be better to just make the entire regex case insensitive instead?

Reply:
yea sg